include/net/wifi_mgmt.h

```diff
@@ -15,6 +15,10 @@
 #include <net/net_mgmt.h>
```

Comment:
Minor issue, but the commit subject could be changed to more appropriate `net: add extern "C" to wifi_mgmt.h`, the include name can be omitted.

Reply:
Fixed, thanks!